docs/navigate.rst

```diff
@@ -140,8 +140,9 @@ View VIP Dashboard
    
       - It is possible to directly click the **Visibility Dashboard** sub-tab and then select the VIP, instead of a redirect from **Visibility Table**.
       
+5. F5 ACI ServiceCenter supports MAC masquerade mac display under BIG-IP endpoint details on VIP visibility dashboard. MAC address type details( eg. MAC masquerade MAC or Normal MAC ) will be displayed on hover over the information icon in the MAC column of BIG-IP endpoint details table.
```

Comment:
(should 'mac' in mac display be capitalized? seems like it.)
... on the VIP visibility dashboard.
MAC address type details (such as MAC masquerade or normal MAC) are displayed when you hover over the Information icon in the MAC column of the BIG-IP endpoint details table.